lib/Basics/win-utils.cpp

```diff
@@ -100,16 +100,11 @@ static void InvalidParameterHandler(const wchar_t* expression,  // expression se
 #endif
 
   LOG_TOPIC("e4644", ERR, arangodb::Logger::FIXME)
-      << "Invalid handle parameter passed"
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
-      << buf;
-
-  std::string bt;
-  TRI_GetBacktrace(bt);
-  LOG_TOPIC("967e6", ERR, arangodb::Logger::FIXME)
-      << "Invalid handle parameter Invoked from: " << bt
+      << "Invalid handle parameter passed: " << buf;
+#else
+      << "Invalid handle parameter passed";
```

Comment:
please log a stacktrace here.

Reply:
This code will only run on Windows, but the CrashHandler can't log stacktraces on Windows.
It is only available for Linux builds.

The whole point of this PR is to remove all other implementation of back/stacktrace functionality that we had elsewhere.

We could improve the CrashHandler so that it logs stacktraces on Windows, too, but I have no idea how to do this and test this myself.